app/models/course.rb

```diff
@@ -417,6 +422,13 @@ def reorder_weeks
     end
   end
 
+  # Makes sure that the home wiki
+  # is always a part of courses wikis.
+  def ensure_home_wiki_in_courses_wikis
+    return if nil?
```

Comment:
What's this for? The implicit `self` receiving `.nil?` would be a `Course` object, so it would never be nil, right?

Reply:
Yup! I migrated this code from controller and forgot this bit.